blocks/components/editable/index.js

```diff
@@ -44,9 +44,9 @@ export default class Editable extends wp.element.Component {
 	onSetup( editor ) {
 		this.editor = editor;
 		editor.on( 'init', this.onInit );
-		editor.on( 'focusout', this.onChange );
 		editor.on( 'NewBlock', this.onNewBlock );
 		editor.on( 'focusin', this.onFocus );
+		editor.on( 'input', this.onChange );
```

Comment:
I'm concerned about the performance impact of binding to `input` in combination with `getContent`, which will become even worse with #466.